### PR TITLE
New assets

### PR DESCRIPTION
**Pull request by matizm**
Added NoText maps
Added Rony and QBU to dictionaries

### CODE REVIEW

File: dictionaries/telemetry/damageCauserName.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
   "BattleRoyaleModeController_Desert_C": "Bluezone",
   "BattleRoyaleModeController_Savage_C": "Bluezone",
   "Boat_PG117_C": "PG-117",
+  "BP_M_Rony_A_01_C": "Pickup Truck (open top)",
+  "BP_M_Rony_A_02_C": "Pickup Truck (open top)",
+  "BP_M_Rony_A_03_C": "Pickup Truck (open top)",
```
Comment (jaloisi834) [inline dictionaries/telemetry/damageCauserName.json]: If they are all open top, you can probably just leave that off.

File: dictionaries/telemetry/vehicle/vehicleId.json [hunk 2]
```diff
@@ -3,6 +3,9 @@
   "AquaRail_A_02_C": "Aquarail",
   "AquaRail_A_03_C": "Aquarail",
   "Boat_PG117_C": "PG-117",
+  "BP_M_Rony_A_01_C": "Pickup Truck (open top)",
+  "BP_M_Rony_A_02_C": "Pickup Truck (open top)",
+  "BP_M_Rony_A_03_C": "Pickup Truck (open top)",
```
Comment (jaloisi834) [inline dictionaries/telemetry/vehicle/vehicleId.json]: Same thing here.

File: dictionaries/telemetry/damageCauserName.json (hunk 1 above)
Comment (jaloisi834) [reply dictionaries/telemetry/damageCauserName.json]: Also, is pickup truck the accepted reference to the rony, or do people call it the rony?

**Comment by jaloisi834** [comment]
I think the new map images should use the data names, and we need to fix the casing (this applies to the existing map images as well which was my bad). Maybe something like this:
```
Miramar_NoText_lowres -> Desert_Main_No_Text_Low_Res
```

File: dictionaries/telemetry/damageCauserName.json (hunk 1 above)
Comment (matizm) [reply dictionaries/telemetry/damageCauserName.json]: It's a pickup truck that's also called the Rony.. Rony might be better here though. I'll update it.

**Comment by matizm** [comment]
Yeah, I didn't like the casing either, but I was trying to be consistent. I'll fix all of them.

**Pull request merged**